spec/index.html

```diff
@@ -293,12 +294,15 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
       </tr>
     </table>
 
-    <p>In some serialization formats it is common to abbreviate <a>IRIs</a>
-      that start with <a>namespace IRIs</a> by using a
-      <a>namespace prefix</a> in order to assist readability. For example, the IRI
+    <p>In some serialization formats it is common
+      to some [=namespace IRIs=] to arbitrary [=namespace prefixes=],
+      and to abbreviate <a>IRIs</a> that start with one of those <a>namespace IRIs</a> by using the corresponding <a>namespace prefix</a>,
+      in order to assist readability.
```

Comment:
```suggestion
    <p>In some serialization formats it is common
      to abbreviate <a>IRIs</a> that start with one of those <a>namespace IRIs</a> by using the corresponding <a>namespace prefix</a>,
      in order to assist readability.
```

Reply:
a verb ("associate") was missing from my text, sorry.
Hopefully, this is now clearer.

The point was to say explicitly that the prefix-IRI association was *specified* in the syntax, not provided by some external convention.
(the previous wording was ambiguous in that respect)